docs/guides/primitives.mdx

```diff
@@ -81,25 +80,46 @@ sampler.run([
+which different providers can derive their own Sampler and Estimator implementations. Among the implementations
+that use Qiskit, you can find reference primitive implementations for local simulation in the [`qiskit.primitives`](/api/qiskit/primitives) module.
+Providers like [Qiskit Runtime](/api/qiskit-ibm-runtime/runtime_service) enable access to appropriate QPUs through native implementations of
+their own primitives.
+
+
+The [`qiskit.primitives`](/api/qiskit/primitives) module enables the development of primitive-style quantum programs and was specifically
+designed to simplify switching between different types of quantum compute resources. As described previously, there are two types of Qiskit primitives; the base class, and their implementations.
+
+### Base class
 
-The `qiskit.primitives` module enables the development of primitive-style quantum programs and was specifically
-designed to simplify switching between different types of quantum compute resources. The module provides three separate classes
-for each primitive type:
+[`BaseEstimatorV2`](/api/qiskit/qiskit.primitives.BaseEstimatorV2) and [`BaseSamplerV2`](/api/qiskit/qiskit.primitives.BaseSamplerV2)
 
-1. `StatevectorSampler` and `StatevectorEstimator`
+These are abstract base classes that define a common interface for implementing primitives. All other classes in the [`qiskit.primitives`](/api/qiskit/primitives) module inherit from these base classes, and developers should use these if they are interested in developing their own primitives-based execution model for a specific provider. These classes might also be useful for those who want to do highly customized processing and find the existing primitives implementations too simple for their needs.
 
-These classes are reference implementations of both primitives and use the simulator built in to Qiskit. They leverage the Qiskit `quantum_info` module in the background, producing results based on ideal statevector simulations.
+### Implementations
 
-2. `BaseSampler` and `BaseEstimator`
+- [`StatevectorEstimator`](/api/qiskit/qiskit.primitives.StatevectorEstimator) and [`StatevectorSampler`](/api/qiskit/qiskit.primitives.StatevectorSampler#statevectorsampler)
 
-These are abstract base classes that define a common interface for implementing primitives. All other classes in the `qiskit.primitives` module inherit from these base classes, and developers should use these if they are interested in developing their own primitives-based execution model for a specific provider. These classes may also be useful for those who want to do highly customized processing and find the existing primitives implementations too simple for their needs.
+These classes are reference implementations of the primitives and use the simulator built into Qiskit. They leverage the Qiskit [`quantum_info`](/api/qiskit/quantum_info#quantum-information) module in the background, producing results based on ideal statevector simulations.
```

Comment:
```suggestion
These classes are reference implementations of the primitives and use the simulator built into Qiskit. They are built with the Qiskit [`quantum_info`](/api/qiskit/quantum_info#quantum-information) module in the background, producing results based on ideal statevector simulations.
```
Rewording because IBM Style says not to use leverage in tech language (and to limit it in marketing content)